i8_terminal/common/layout.py

```diff
@@ -54,9 +55,16 @@ def df2Table(df: DataFrame, style_profile: str = "default", columns_justify: Dic
         "Revenue Beat Rate": "right",
         "EPS Surprise": "right",
         "Revenue Surprise": "right",
+        "EPS Consensus": "right",
+        "Revenue Consensus": "right",
+        "Eps Surprise": "right",
     }
     for c in df.columns:
-        table.add_column(c, justify=columns_justify.get(c, default_justify.get(c, "left")))
+        table.add_column(
+            c,
+            justify=columns_justify.get(c, default_justify.get(c, "left")),
+            min_width=min(max(df[c].str.len().max(), len(df[c].name)), MAX_COL_LENGTH),
```

Comment:
The logic or naming is confusing. If `min_width` is supposed to be a minimum value, it means the column length can be more than what we set here. If that is the case, then we could have a width that is greater than `MAX_COL_LENGTH`. In that case, the naming of `MAX_COL_LENGTH` is confusing. Please think about it, and make sure it works as expected.